ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/restore/view/RestoreView.kt

```diff
@@ -321,11 +340,23 @@ private fun Autocomplete(
 
         LazyRow(highlightModifier.testTag(RestoreTag.AUTOCOMPLETE_LAYOUT)) {
             items(it) {
-                Button(
-                    modifier = Modifier.testTag(RestoreTag.AUTOCOMPLETE_ITEM),
-                    onClick = { onSuggestionSelected(it) }
+                Surface( // TODO create chip
```

Comment:
What's this TODO comment for?  Is there a followup issue you want to file for this?

Reply:
no sorry, I forgotten about this. I will fix it